HW_1/task_1.py

```diff
@@ -0,0 +1,18 @@
+# Первая задача из домашнего задания.
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# *Пример:*
+
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+n = int(input())  # считываем трехзначное число
+sum_of_digits = 0  # инициализируем сумму цифр числа
+
+# вычисляем сумму цифр числа
+while n > 0:
+    digit = n % 10  # получаем последнюю цифру числа
+    sum_of_digits += digit  # добавляем цифру к сумме
```

Comment:
Это можно совместить
```suggestion
    sum_of_digits += n % 10# добавляем цифру к сумме
```